config.py

```diff
@@ -57,6 +60,8 @@ class Config(object):
     ACCESS_CASE = 'surveys/access-case'
     DOWNLOAD_CI = 'surveys/download-ci'
     UPLOAD_CI = 'surveys/upload-ci'
+    ADD_SURVEY = 'surveys/add-survey'
+    CONFIRM_ADD_ORGANISATION_SURVEY = 'surveys/add-survey/confirm-add-organisation-survey'
```

Comment:
This is fine right now because it's consistent with the rest of the repo but I think it'd be good to move these out of `config.py` in the future

Reply:
Yeah I agree